adafruit_gc_iot_core.py

```diff
@@ -120,25 +146,31 @@ def disconnect(self):
         # De-initialize MiniMQTT Client
         self._client.deinit()
 
-    def reconnect(self):
+    def reconnect(self) -> None:
         """Reconnects to the Google MQTT Broker."""
         try:
             self._client.reconnect()
         except Exception as err:
             raise MQTT_API_ERROR("Error reconnecting to Google MQTT.") from err
 
-    def connect(self):
+    def connect(self) -> None:
         """Connects to the Google MQTT Broker."""
         self._client.connect()
         self._connected = True
 
     @property
-    def is_connected(self):
+    def is_connected(self) -> bool:
         """Returns if client is connected to Google's MQTT broker."""
         return self._connected
 
     # pylint: disable=not-callable, unused-argument
-    def _on_connect_mqtt(self, client, userdata, flags, return_code):
+    def _on_connect_mqtt(
+        self,
+        client: "MQTT.MQTT",
+        user_data: Optional[Any],
+        flags: bytearray,
```

Comment:
Is there somewhere you found `bytearray`?  I was able to find what looks like `int` in the base library for this:

https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/5f783d5fe46b7067f0bc172773c6cb1a04c6d013/adafruit_minimqtt/adafruit_minimqtt.py#L525

Reply:
Fixed!